tests/bulk/README.md

```diff
@@ -1,192 +0,0 @@
-## Bulk import testing
```

Comment:
Where did this go?

Reply:
I felt like it wasn't needed to be kept in the repository, so I moved it to a [gist](https://gist.github.com/matthewhegarty/0f4d5ccaddcb6a2341744ca9afef4bcd).  I have updated `testing.rst` to define the procedure for bulk testing.

Reply:
I found it interesting to have the statistics at hand, never got to use bulk but it is on the radar. I won't argue about removing this if you want to clean it up :)